modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -28,9 +28,29 @@ class AppealsApi::V2::DecisionReviews::NoticeOfDisagreementsController < Appeals
 
   def create
     @notice_of_disagreement.save
+    AppealsApi::PdfSubmitJob.perform_async(
+      @notice_of_disagreement.id,
+      'AppealsApi::NoticeOfDisagreement',
+      API_VERSION
```

Comment:
I'll preface this by saying I recognize it sounds like I'm splitting hairs, but I think it's important that we distinguish between our api versions and the pdf generator versions. So far, we've had them in sync with each other (api v2 == pdf v2), but that's not always guaranteed. 

It could be that there's a v3 pdf we'll need to make where none of the form_data changes but the fields move around - in that case using API_VERSION would be incorrect. I think we should just pass the version string directly into the job argument in this case.

Reply:
Not splitting. I had the same thought and went in the lamer direction. I'll get it fixed up.